doc/user_guide/transformations.rst

```diff
@@ -755,6 +755,7 @@ transformations currently supported allow the addition of:
 
 * an **OpenMP Parallel** directive
 * an **OpenMP Target** directive
+* an **OpenMP Declare Target** directive
```

Comment:
Please could you also update the AvailableTransformations section with a `.. autoclass` directive for this new transformation (in alphabetical order with all the others).